### PR TITLE
dev-only: Happy new year and make release checklist a top-to-bottom todolist instead of separate sections

### PR DESCRIPTION
**Pull request by Garandor**
Only two files touched:
- release issue template
- crate_draft_releases.yml workflow

All other changes are just copyright updates

release issue: 
- Redone the sequence so we don't distinguish between client/runtime/all releases.
- Now just follow top-to-bottom and do the step when appropriate

draft_releases:
- Only triggers on git tags that start with a `v`

Signed-off-by: Adam Reif <Garandor@manta.network>

## Description

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari`, `A-dolphin` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.
- If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. You can run the `metadata_diff.yml` workflow for help. If this number is updated, then the `spec_version` must also be updated
- Verify benchmarks & weights have been updated for any modified runtime logics

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -74,12 +65,6 @@ Note: Do not publish draft releases from PR branches, because those branches wil
   - [ ] Subscan team. Ensure subscan service can continue to scan calamari blocks.
 
 ## Notes
-
-### Burn In
-
-- [ ] Ensure that Manta DevOps has run the new release on Baikal nodes for at least 3 days prior to publishing the release.
-- [ ] Check all testnet nodes [on Grafana](https://grafana.pulse.pelagos.systems/d/rYdddlPWk/node-exporter-full?orgId=1&refresh=1m&var-DS_PROMETHEUS=default&var-job=calamari-testnet%20invulnerable%20collator%20(ssl)&var-node=crunchy.baikal.testnet.calamari.systems:443&var-diskdevices=%5Ba-z%5D%2B%7Cnvme%5B0-9%5D%2Bn%5B0-9%5D%2B%7Cmmcblk%5B0-9%5D%2B) for irregularities since the upgrade
-
```
Comment (ghzlatarev) [inline .github/ISSUE_TEMPLATE/release.md:82]: Why is this removed, previous sections link to it.

File: .github/ISSUE_TEMPLATE/release.md [hunk 2]
```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
...
 - [ ] Check that the new client and/or runtime versions have [burned-in](#burn-in) without issue for at least 3 days.
-- [ ] Before declaring a successful burn-in make sure to check for anomalies in our nodes' memory, cpu, disk and network usage via the [Grafana Node Explorer](https://grafana.pulse.pelagos.systems/d/rYdddlPWk/node-exporter-full). These would include but are not limited to: memory leaks, cpu spikes, spike in tcp sockets waiting to close, etc. Make sure to take a look at all of the available graphs, because some problems might only show up in the collapsed views.
 - [ ] Keep an eye out on the [manta status dashboard](https://status.manta.network/) for additional metrics like outages.
 
-Note: Do not publish draft releases from PR branches, because those branches will be deleted when the PR is merged.
```
Comment (ghzlatarev) [inline .github/ISSUE_TEMPLATE/release.md:64]: why is this note removed

File: .github/ISSUE_TEMPLATE/release.md [hunk 3]
```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
...
+- [ ] Coordinate with marketing team for documentation updates and other relevant tasks.
+- [ ] Monitor [Grafana Node Explorer](https://grafana.pulse.pelagos.systems/d/rYdddlPWk/node-exporter-full) for anomalies in our nodes' memory, cpu, disk and network usage. These would include but are not limited to: memory leaks, cpu spikes, spike in tcp sockets waiting to close, etc. Make sure to take a look at all of the available graphs because some problems might only be visible in views that are collapsed by default
 - [ ] Check that the new client and/or runtime versions have [burned-in](#burn-in) without issue for at least 3 days.
-- [ ] Before declaring a successful burn-in make sure to check for anomalies in our nodes' memory, cpu, disk and network usage via the [Grafana Node Explorer](https://grafana.pulse.pelagos.systems/d/rYdddlPWk/node-exporter-full). These would include but are not limited to: memory leaks, cpu spikes, spike in tcp sockets waiting to close, etc. Make sure to take a look at all of the available graphs, because some problems might only show up in the collapsed views.
```
Comment (ghzlatarev) [inline .github/ISSUE_TEMPLATE/release.md:61]: why is this removed

File: .github/ISSUE_TEMPLATE/release.md [hunk 4]
```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
...
 - [ ] Complete the [manual QA workflow](https://www.notion.so/mantanetwork/d55be01354bb4f579b16d6e34df9e2e1?v=dcfa54e2b4a343ad9b899574ddb94a1c).
-- [ ] Verify Polkadot JS API are up to date with the latest
-    runtime changes.
-- [ ] Execute runtime upgrade to Baikal relay and verify network stability.
-- [ ] Execute runtime upgrade to Calamari @ Baikal and verify network stability.
-- [ ] Execute runtime upgrade to Calamari @ Moonbase-Relay and verify network stability.
-- [ ] Execute runtime upgrade to Dolphin @ Baikal and verify network stability.
-- [ ] Check network health metrics like average block times, block authors, etc with this tool https://parachain-utilities.vercel.app/
-
-Note: Usually update client first then runtime.
```
Comment (ghzlatarev) [inline .github/ISSUE_TEMPLATE/release.md:40]: why is this note removed

**Comment by Dengjianping** [comment]
Can you update copyright year in this pr?
```
// Copyright 2020-2023 Manta Network.
```

File: .github/ISSUE_TEMPLATE/release.md [hunk 5]
```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
...
+    - [ ] Generate new changelog using `dev-tools` repo
+    - [ ] Merge when green and reviewed
+    - [ ] Tag the release
+    - [ ] Check that a draft release has been created at https://github.com/Manta-Network/Manta/releases with relevant [release notes](#release-notes)
+    - [ ] Check that build artifacts have been added to the
+        draft-release
+    - [ ] Create a Pre-Release on github and add Release Notes (see below for what needs to be in them)
+
+# Deploy to internal testnets ( fast runtime )
+    - [ ] Verify Polkadot JS API are up to date with the latest
```
Comment (Dengjianping) [inline .github/ISSUE_TEMPLATE/release.md]: These items are not rendered correctly.

File: .github/ISSUE_TEMPLATE/release.md [hunk 6]
```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
...
-    https://github.com/Manta-Network/Manta/releases with relevant [release
-    notes](#release-notes)
-- [ ] Check that build artifacts have been added to the
-    draft-release
-- [ ] Coordinate with marketing team for documentation updates and other relevant tasks.
-- [ ] Update changelog.
 - [ ] If the release contains any changes that break/change functionality used in https://github.com/Manta-Network/sdk (e.g. RPC changes, see also [extrinsic ordering](#extrinsic-ordering)), raise a PR there and **block this release** until your PR has been merged and incorporated in a new SDK release.
+
+# Deploy to public testnet
+    - [ ] execute client upgrade on calamari @ moonbase-relay nodes if needed
```
Comment (Dengjianping) [inline .github/ISSUE_TEMPLATE/release.md]: These items are not rendered correctly as well.

**Review by ferrell-code — COMMENTED** [review]
Not in diff, but the Security Audit section always fails.

Not sure the best solution as I agree with the check, maybe just leave it as is.

File: .github/ISSUE_TEMPLATE/release.md (hunk 4 above)
Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md:40]: because update client is now an explicit step before update runtime

File: .github/ISSUE_TEMPLATE/release.md (hunk 2 above)
Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md:64]: draft releases only get created on manta branch, we changed the CI

File: .github/ISSUE_TEMPLATE/release.md (hunk 1 above)
Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md:82]: The explanations here are duplicated with the instructions above, i'll remove the link

File: .github/ISSUE_TEMPLATE/release.md (hunk 3 above)
Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md:61]: duplicated with line 52

File: .github/ISSUE_TEMPLATE/release.md (hunk 5 above)
Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md]: thanks, fixing it

**Comment by Garandor** [comment]
> Not in diff, but the Security Audit section always fails.
> 
> Not sure the best solution as I agree with the check, maybe just leave it as is.

fixing this should be in scope for 4.0.1

File: .github/ISSUE_TEMPLATE/release.md (hunk 2 above)
Comment (ghzlatarev) [reply .github/ISSUE_TEMPLATE/release.md:64]: Can't it be triggered also from a tag (which you can do from the dev branch), there are 3 ORs - [qa9QA7HC6SR82Tbsq2PSn9xQ3sG56Q3ToUENZt5bLvRJ7Rf](https://github.com/Manta-Network/Manta/blob/manta/.github/workflows/publish_draft_releases.yml#L36)
That's the main thing this line is avoiding.

File: .github/ISSUE_TEMPLATE/release.md (hunk 3 above)
Comment (ghzlatarev) [reply .github/ISSUE_TEMPLATE/release.md:61]: Cool, didn't see it was moved.

File: .github/ISSUE_TEMPLATE/release.md (hunk 4 above)
Comment (ghzlatarev) [reply .github/ISSUE_TEMPLATE/release.md:40]: Ok, I'd make a note then that sometimes (rarely) runtime might have to be updated first.

File: .github/ISSUE_TEMPLATE/release.md [hunk 7]
```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
...
+    - [ ] Check that build artifacts have been added to the
+        draft-release
+    - [ ] Create a Pre-Release on github and add Release Notes (see below for what needs to be in them)
+
+# Deploy to internal testnets ( fast runtime )
 - [ ] Verify Polkadot JS API are up to date with the latest
     runtime changes.
-- [ ] Execute runtime upgrade to Baikal relay and verify network stability.
+- [ ] Execute client upgrade on Baikal relaychain nodes if needed
+- [ ] Execute runtime upgrade to Baikal relaychain and verify network stability.
```
Comment (ghzlatarev) [inline .github/ISSUE_TEMPLATE/release.md]: Rename Baikal to kusama-internal or kusama-staging maybe

Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md]: Kusama internal is the public testnet.

I'm counting on us relaunching a fastamari-on-baikal as internal testnet soonTM

Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md]: Or are you planning on fully retiring the Baikal relay and running everything on kusama-internal (which also needs to be renamed now)

Damn, our naming got SNAFU'd by the dol v3 release. I'm wondering now if we should just keep `kusama-internal` as internal testnet for the future and release a fully new network for the next public testnet without Alice-Eve keys on any attached chain

File: .github/ISSUE_TEMPLATE/release.md (hunk 4 above)
Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md:40]: Added

File: .github/ISSUE_TEMPLATE/release.md (hunk 2 above)
Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md:64]: I think the real fix here is to only run this workflow when it's run from a tag that starts with `v`
Now that this doesn't run any tests, there's no real reason to run it as part of release branches or on manta branch anyway.

I fixed it by getting rid of the ifs and explaining the process

**Review by ferrell-code — DISMISSED** [review]

**Review by ghzlatarev — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/release.md (hunk 7 above)
Comment (ghzlatarev) [reply .github/ISSUE_TEMPLATE/release.md]: I mean whatever new pre-staging we deploy (fast or not) it would also be on the same relay chain, which is not called Baikal anymore, but Kusama-internal.

Comment (Garandor) [reply .github/ISSUE_TEMPLATE/release.md]: Are you sure? This would mean we have to have a customized runtime with changed paraID just for this deployment.

Not sure saving 3 compute nodes to run 15 validators on is worth that, deploying the network is fast with ansible and we can reset the pre-staging relay whenever sync time gets annoying

**Review by ferrell-code — APPROVED** [review]
